Week 1/01/Q10-01-ofelia.c

```diff
@@ -0,0 +1,26 @@
+#include<stdio.h>
+int main()
+{
+  int n,i,c=1;
+  printf("Enter a number");
+  scanf("%d",&n);
+  for(i=2;i<=n/2;i++)
```

Comment:
Good use of the for loop. 
Program can be made more efficient by changing n/2 by sqrt(n)